runtime/vm/rasdump.c

```diff
@@ -289,7 +293,12 @@ configureRasDump(J9JavaVM *vm)
 void
 J9RASInitialize(J9JavaVM* javaVM)
 {
+#if defined(OSX)
+	char **environ = *_NSGetEnviron();
+#else
```

Comment:
Bit of a nitpick - usually we label the else as well
```suggestion
#else /* defined(OSX) */
```